### PR TITLE
Add params in gradle.properties

### PR DESCRIPTION
**Pull request by Goooler**
- Add params to enhance the speed of build

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -1,14 +1,26 @@
-org.gradle.jvmargs=-Xmx4g
+org.gradle.caching=true
```
Comment (colinrtwhite) [inline gradle.properties:1]: I think most of these are enabled by default.

File: gradle.properties [hunk 2]
```diff
@@ -1,14 +1,26 @@
...
+org.gradle.configureondemand=true
+org.gradle.jvmargs=-Xmx4g -Dfile.encoding=UTF-8
+org.gradle.parallel=true
 
 # AndroidX
 android.useAndroidX=true
 
+# Enable install debug apk manually
+android.injected.testOnly=false
```
Comment (colinrtwhite) [inline gradle.properties:10]: What does this do?

File: gradle.properties [hunk 3]
```diff
@@ -1,14 +1,26 @@
...
+
 # Disable welcome message.
 systemProp.org.gradle.internal.launcher.welcomeMessageEnabled=false
 
 # Disable adding the standard library by default.
 kotlin.stdlib.default.dependency=false
 
+# Enhance speed of kotlin build
+kotlin.code.style=official
```
Comment (colinrtwhite) [inline gradle.properties:19]: How does enabling the official kotlin code style improve build speed?

File: gradle.properties [hunk 4]
```diff
@@ -1,14 +1,26 @@
...
+kotlin.incremental=true
```
Comment (colinrtwhite) [inline gradle.properties:20]: These are all enabled by default.

File: gradle.properties (hunk 3 above)
Comment (Goooler) [reply gradle.properties:19]: I was negligent, it should be separated

File: gradle.properties (hunk 4 above)
Comment (Goooler) [reply gradle.properties:20]: image
Default value is false

File: gradle.properties (hunk 2 above)
Comment (Goooler) [reply gradle.properties:10]: Could let debuggale apk installed manually

File: gradle.properties (hunk 1 above)
Comment (Goooler) [reply gradle.properties:1]: Not sure this properity is enabled by default, bucause not found default value in gradle source

File: gradle.properties (hunk 4 above)
Comment (colinrtwhite) [reply gradle.properties:22]: Hmm according to [this blog](https://blog.jetbrains.com/kotlin/2017/03/kotlin-1-1-1-is-out/) it was enabled by default in Kotlin `1.1.1`.

File: gradle.properties (hunk 2 above)
Comment (colinrtwhite) [reply gradle.properties:10]: How does that increase build speed?

Comment (Goooler) [reply gradle.properties:10]: No increasement, just convient for test

File: gradle.properties (hunk 4 above)
Comment (Goooler) [reply gradle.properties]: Ok, removed

**Comment by colinrtwhite** [comment]
Thanks, however I'd prefer to stick to the defaults to prioritize stability over build speed.

**Pull request closed without merging**